zerver/lib/outgoing_webhook.py

```diff
@@ -157,6 +158,17 @@ def fail_with_message(event, failure_message):
     failure_message = "Failure! " + failure_message
     send_response_message(event['user_profile_id'], event['message'], failure_message)
 
+def notify_bot_owner(event, notification_message):
+    # type: (Dict[str, Any], Text) -> None
+    bot_id = event['user_profile_id']
+    bot_user = get_user_profile_by_id(bot_id)
+    message_info = {'display_recipient': [{'email': bot_user.email}],
+                    'type': 'private'}
+    for bot_owner_id in bot_owner_userids(bot_user):
```

Comment:
We definitely just wanted `user_profile.bot_owner`; the `bot_owner_userids` has the actual owner of the bot plus organization admins.  We don't want to message all organization admins about someone's buggy bot!

Reply:
I think fixing this will make `message_info` a simple declaration and probably resolve the mypy problem.